.github/workflows/pr_security_test_one.yml

```diff
@@ -49,6 +49,14 @@ on:
           - '14.21.3'
           - '16.20.1'
         default: '16.20.1'
+      DB_TYPE:
```

Comment:
Couldn't we keep the same variable everywhere for consistency? It's a bit hard to follow between DB_TYPE, DATABASE, DB_SERVICE, DB_SERVER 🤪